### ISSUE TITLE
Weights for MIF-ST and MIF-GVP

### ISSUE THREAD

**Issue by twidatalla** [microsoft/protein-sequence-models#15]
Hello, I see the pre-trained MIF-ST model has input node features of size [node features + alphabet size] not [node features + CARP embedding size]. So I'm assuming this means you guys concatenated the logits (not corrupted/masked one-hot encodings) at each position from CARP to the node features as your input?

Also, is the MIF-GVP model different from the pretrained model for Inverse folding in the GVP Paper? I see you guys got higher sequence recovery than them so I wasn't sure. If so would you be able to upload the weights for that model as well?

Great Paper by the way! As someone working on training multi-modal models for protein property prediction I was very excited to find a pretrained structure+language model as previously I could only find one or the other.

**Comment by yangkky** [comment]
Yes, mif-st uses carp logits.

MIF-GVP is trained from scratch as well with the Bert objective. I'll
upload the weights to zenodo soon.

**Comment by yangkky** [comment]
I've pushed my GVP implementation (`sequence_models/gvp.py`) and uploaded the model weights to Zenodo. If it'd be useful, I can make it loadable the same way MIF and MIF-ST are, but that will take me a little longer. 

Note that our recovery is higher because we're using a BERT-style masked language model task and not an autoregressive language modeling task.

**Comment by twidatalla** [comment]
Thank you Kevin these are a great help. The load-ability is fine no worries about that. The only thing which needs a bit of clarification are the params for gvp (node_in_dim, node_h_dim, edge_in_dim, edge_in_dim). I was able to infer them by looking at the dimensionality of the layers in the mifgvp state dict from zenodo but it might be nice just to include them as defaults in the __init__ function in gvp.py as they're different than the original paper.

 def __init__(self, node_in_dim, node_h_dim,
                 edge_in_dim, edge_h_dim, num_layers=3, drop_rate=0, vocab_size=30)
